plugins/linuxv2/model/l3/keys.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package l3
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+)
+
+const (
+	/* ARP */
+
+	// staticArpKeyPrefix is a prefix used in ETCD to store configuration for Linux static ARPs.
+	staticArpKeyPrefix = "linux/config/v2/arp/"
+
+	// staticArpKeyTemplate is a template for key representing Linux ARP entry configuration.
+	staticArpKeyTemplate = staticArpKeyPrefix + "{if}/{ip}"
+
+	/* Route Config */
+
+	// staticRouteKeyPrefix is a prefix used in ETCD to store configuration for Linux static routes.
+	staticRouteKeyPrefix = "linux/config/v2/route/"
+
+	// staticRouteKeySuffix is a suffix common to all keys representing routes.
+	staticRouteKeySuffix = "{dest-net}/{dest-mask}/{out-intf}"
+
+	// staticRouteKeyTemplate is a template for key representing Linux Route configuration.
+	staticRouteKeyTemplate = staticRouteKeyPrefix + staticRouteKeySuffix
+
+	/* Link-local route (derived) */
+
+	// staticLinkLocalRouteKeyPrefix is a prefix for keys derived from link-local routes.
+	staticLinkLocalRouteKeyPrefix = "linux/link-local-route/"
+
+	// staticLinkLocalRouteKeyTemplate is a template for key derived from link-local route.
+	staticLinkLocalRouteKeyTemplate = staticLinkLocalRouteKeyPrefix + staticRouteKeySuffix
+)
+
+/* ARP */
+
+// StaticArpKeyPrefix returns the prefix used in ETCD to store config for Linux static ARPs
+func StaticArpKeyPrefix() string {
```

Comment:
In the VPP plugin we have already got rid of these *helper functions* which return constatns. All prefixes should be exported so they can be used directly and leave only functions which concatenate keys. 

Aim for less code ☀️